pkg/convert/xmb.go

```diff
@@ -0,0 +1,71 @@
+package convert
+
+import (
+	"encoding/xml"
+	"fmt"
+
+	"go.expect.digital/translate/pkg/model"
+	"golang.org/x/text/language"
+)
+
+type xmb struct {
+	XMLName  xml.Name     `xml:"messagebundle"`
+	Language language.Tag `xml:"srcLang,attr"`
+	Messages []msg        `xml:"message"`
+}
+
+type msg struct {
+	ID          string `xml:"id,attr"`
+	Description string `xml:"desc,attr"`
+	Content     string `xml:"content"`
+	Fuzzy       bool   `xml:"fuzzy,attr"`
+}
+
+func FromXMB(b []byte) (model.Messages, error) {
+	var xmbFormat xmb
+
+	err := xml.Unmarshal(b, &xmbFormat)
+	if err != nil {
+		return model.Messages{}, fmt.Errorf("unmarshaling xmb to model.Messages: %w", err)
+	}
+
+	messages := model.Messages{Messages: make([]model.Message, 0, len(xmbFormat.Messages))}
```

Comment:
Why use different implementations, reserving slice capacity in one and length in other.
`	xmbFormat := xmb{
		Messages: make([]msg, len(m.Messages)),
		Language: m.Language,
	}`